.codeclimate.yml

```diff
@@ -0,0 +1,21 @@
+# .codeclimate.yml
+version: "2"
+checks:
+  # Disable checks for similar code.
+  # Because this creates too many false positives.
+  similar-code:
+    enabled: false
+  # Increase the number of lines allowed for a method.
+  # Because this is more reasonable for JavaScript.
+  method-lines:
```

Comment:
Wouldn't this trigger on every component since they are each functions?

Reply:
this is the boilerplate given to us on the notion document, it's a starting point.